### ISSUE TITLE
#640 introduces monitoring problems

### ISSUE THREAD

**Issue by nickolaev** [networkservicemesh/networkservicemesh#669]
After merging #640 "Issue #622: Properly handle NSC/NSE dies (#640)" the monitoring for skydive is not behaving as expected. Reverting 605d99b8e2f1afc4f538a08d38250db5d0fb86b9 solves the problems and retruns back skydive to its normal operation.

**Comment by nickolaev** [comment]
@lobkovilya, please check. @matrohon can provide more info if needed.

**Comment by matrohon** [comment]
here is the way to reproduce this bug:

```
make vagrant-start
make k8s-save
make k8s-infra-deploy
make k8s-vpn-deploy
```

**Comment by edwarnicke** [comment]
@matrohon Was this closed by #673 ?

**Comment by matrohon** [comment]
There is still an issue with skydive that doesn't display crossconnects properly after skydive is restarting. Let me test further to see if it comes from the nsmd monitoring stack or from the skydive probe.

**Comment by matrohon** [comment]
I've fixed the skydive bug. Still, something is weird with the crossconnect-monitor container: when it starts and no crossconnect exists, it doesn't logs empty "INITIAL_STATE_TRANSFER" messages. @lobkovilya Is it expected?

**Comment by matrohon** [comment]
#691 aims at fixing this "issue"

**Comment by edwarnicke** [comment]
Merged #691, closing.
